server/src/main/kotlin/suwayomi/tachidesk/graphql/mutations/DownloadMutation.kt

```diff
@@ -25,7 +25,7 @@ class DownloadMutation {
         val chapters: List<ChapterType>
     )
 
-    fun deleteChapters(input: DeleteChapterDownloadsInput): DeleteChapterDownloadsPayload {
+    fun deleteChapterDownloads(input: DeleteChapterDownloadsInput): DeleteChapterDownloadsPayload {
```

Comment:
what about `deleteDownloadedChapter(s)`, `DeleteDownloadedChapter(s)Input` and `DeleteDownloadedChapter(s)Payload`?

Reply:
Good idea